ecommerce/frontend/src/components/product_card/ProductCard.js

```diff
@@ -1,33 +1,65 @@
 import React from 'react'
 import { Button, Rate } from 'antd'
-import { PlusCircleOutlined, HeartOutlined } from '@ant-design/icons'
+import { PlusCircleOutlined, HeartOutlined, EditOutlined, DeleteOutlined } from '@ant-design/icons'
 import { Link } from 'react-router-dom'
 import './ProductCard.less'
 import { useAppContext } from '../../context/AppContext'
 import { round, truncate } from '../../utils'
+import { ISO_8601 } from 'moment'
 
-export const ProductCard = ({ product, width = 350 }) => {
+export const ProductCard = ({ product, width = 350, editMode = false }) => {
     const { addShoppingCartItem } = useAppContext()
 
     const onAddToCart = product => {
         addShoppingCartItem(product, 1)
     }
 
+    const onDeleteProductCard = () => {
+        console.log('delete product card')
+    }
+    const onEditProductCard = () => {
+        console.log('edit product card')
+    }
     const onAddToList = product => {}
 
-    const { title, rating, price, price_after_discount, currency, images, id } = product
-
-    // const discountPrice = price * (1 - discount)
+    const { title, rating, price, price_after_discount, currency = '₺', images, id, vendor } = product
```

Comment:
Can you make currency TL instead just like we used everywhere?

Reply:
sure